dev_guide/events.adoc

```diff
@@ -45,3 +45,172 @@ This section will be expanded as more information becomes available
 
 // TODO: out-of-band example
 ////
+
+[[events-reference]]
+== Comprehensive List of Events
+
+// TODO:
+// - Consult various source code; create tables.
+//   - Kubernetes
+//     pkg/kubelet/container/event.go
```

Comment:
This is going to be a pain to keep up to date. From what I understand, events are not guaranteed to remain backwards compatible, so they could change at any time.